src/components/modals/Modal.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactModal from 'react-modal';
+
+import FAIcon from 'components/FAIcon';
+import {FormattedMessage} from "react-intl";
+import {getBEMClassName} from "../../utils";
```

Comment:
```suggestion
import {getBEMClassName} from 'utils';
```
please use absolute imports